src/routes/About/About.js

```diff
@@ -0,0 +1,7 @@
+export default function About(){
+    return(
+        <div>
+            <h2>About</h2>
+        </div>
+    );
+}
```

Comment:
For components that don't have any testable functionality we'll usually put in a simple render test just to make sure it doesn't have any errors:
`test('renders successfully', () => {
  render(
      <About />
  );
});`